ci-operator/step-registry/cucushift/hypershift-extended/enable-mgmt/cucushift-hypershift-extended-enable-mgmt-commands.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -euo pipefail
+
+if [ -f "${SHARED_DIR}/cluster-type" ] ; then
+    CLUSTER_TYPE=$(cat "${SHARED_DIR}/cluster-type")
+    if [[ "$CLUSTER_TYPE" == "osd" ]] || [[ "$CLUSTER_TYPE" == "rosa" ]]; then
+        echo "Skip the step. The managed clusters generate the testing accounts by themselves"
+        exit 1
```

Comment:
why skip it when cluster type is osd or rosa ? I thought it is for osd/rosa clusters